bindings/py/cpp_src/bindings/algorithms/py_SDR.cpp

```diff
@@ -255,11 +255,30 @@ special, it is replaced with the system time.  The default seed is 0.)",
         }));
 
 
-        py::class_<SDR_Proxy, SDR> py_Proxy(m, "SDR_Proxy");
+        py::class_<SDR_Proxy, SDR> py_Proxy(m, "SDR_Proxy",
+R"(SDR_Proxy presents a view onto an SDR.
```

Comment:
what is this R" supposted to be? some formating?

Reply:
It's the start of a multi-line string.
* Starts with `R"(`
* Ends with `)"`  Yes, the parenthesis go inside of the quotes
* Anything inside, arbitrary length, no need to backslash escape line endings.

I think they added this is C++11? Its great, right?